src/internal/_isIterable.js

```diff
@@ -0,0 +1,5 @@
+var _symIterator = require('./_symIterator');
+
+module.exports = function _isIterable(x) {
+    return x[_symIterator] || (x.next && typeof x.next === 'function');
+};
```

Comment:
Do we want to return a Boolean? This may not be necessary for an internal function.